tests/data/schema.gql

```diff
@@ -398,7 +398,7 @@ type Query {
     ids: [GlobalID!]!
   ): [StaffType!]!
   issueList: [IssueType!]!
-  milestoneList(filters: MilestoneFilter, order: MilestoneOrder, pagination: OffsetPaginationInput): [MilestoneType!]!
```

Comment:
I don't know what happened here but this was copied from a previous commit, tests were failing otherways.

Reply:
There was a change in the inheritance order in the `strawberry-graphql-django` that made that change happpen

Run `poetry install` so that you have the latest version available, it should fix this.

Reply:
Ok

Reply:
restored

Reply:
There are still some changes in this file, which shouldn't have

Reply:
Weird, I'll fix that.